lib/matplotlib/pyplot.py

```diff
@@ -2486,24 +2499,26 @@ def getname_val(identifier):
 def _autogen_docstring(base):
```

Comment:
Doesn't that function become obselete? The only thing it did is append the kwarg information on hold.